scripts/docker-create-push-manifests.sh

```diff
@@ -18,7 +18,7 @@ docker manifest create "${img_march}" \
 docker manifest push "${img_march}"
 
 # PUSH latest if it is a release build
-if [ "$IsPushLatest" = "true" ]; then
+if [ "$IsPushLatest" = "true" -a "${IS_ELIXIR:-no}" = "no" ]; then
```

Comment:
maybe can keep this script more generic if we pass `$2` as `false` from where it's invoked.